src/discussions/topics/data/slices.js

```diff
@@ -1,15 +1,17 @@
 /* eslint-disable no-param-reassign,import/prefer-default-export */
```

Comment:
if you will export default in constants file you wont have to suppress prefer default export here